static/src/js/schemas/uiSchemas/services/serviceContacts.js

```diff
@@ -0,0 +1,690 @@
+                          },
+                          {
+                            'ui:col': {
+                              md: 6,
+                              children: ['Value']
+                            }
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              ]
+            },
+            Type: {
+              'ui:widget': CustomSelectWidget
+            }
+          }
+        },
+        Addresses: {
+          'ui:title': 'Contact Address',
+          items: {
+            'ui:field': 'layout',
+            'ui:layout_grid': {
+              'ui:row': [
+                {
+                  'ui:col': {
+                    md: 12,
+                    children: [
+                      {
+                        'ui:row': [
+                          {
+                            'ui:col': {
+                              md: 6,
+                              children: ['Country']
+                            }
+                          }
+
+                        ]
+                      },
+                      {
+                        'ui:group': 'Street Addresses',
+                        'ui:group-classname': 'h3-title',
+                        'ui:group-box-classname': 'h2-box',
+                        'ui:row': [
+                          {
+                            'ui:col': {
+                              md: 12,
+                              children: ['StreetAddresses']
+                            }
+                          }
+                        ]
+                      },
+                      {
+                        'ui:row': [
+                          {
+                            'ui:col': {
+                              md: 4,
+                              children: ['City']
+                            }
+                          },
+                          {
+                            'ui:col': {
+                              md: 4,
+                              children: ['StateProvince']
+                            }
+                          },
+                          {
+                            'ui:col': {
+                              md: 4,
+                              children: ['PostalCode']
+                            }
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              ]
+            },
+            StreetAddresses: {
+              'ui:field': 'streetAddresses'
+            },
+            StateProvince: {
+              'ui:title': 'State / Province'
+            },
+            PostalCode: {
+              'ui:title': 'Postal Code'
+            },
+            Country: {
+              'ui:widget': CustomCountrySelectWidget
+            }
+          }
+        },
+        OnlineResources: {
+          'ui:title': 'Online Resources',
```

Comment:
'ui:header-level': 'h4' doesn't seem to work here, but this header is very small

Reply:
Adding 'ui:header-level':'h4' here worked